tools/cell_census_builder/validate.py

```diff
@@ -447,6 +478,44 @@ def validate_manifest_contents(assets_path: str, datasets: List[Dataset]) -> boo
     return True
 
 
+def validate_consolidation(soma_path: str, experiment_builders: List[ExperimentBuilder]) -> bool:
+    """Verify that obs, var and X layers are all fully consolidated & vacuumed"""
```

Comment:
this code should walk all elements the collections, not just the pre-defined names, and check that _any_ DataFrame/NDArray is consolidated.

In other words, do something like:
```python
for name in collection:
    if isinstance(collection[name], (soma.DataFrame, soma.SparseNDArray, soma.DenseNDArray):
        # check fragment count
    else:
        # recursively call on collection[name]
```

In other words, check everything, and don't hardwire in magic names as we will change them over time.